runtime-modules/content/src/lib.rs

```diff
@@ -2356,25 +2361,27 @@ decl_module! {
             origin,
             channel_id: T::ChannelId,
             actor: ContentActor<T::CuratorGroupId, T::CuratorId, T::MemberId>,
-            new_transfer_status: ChannelTransferStatus<T::MemberId, T::CuratorGroupId, BalanceOf<T>>
+            candidate_status: ChannelTransferStatus<T::MemberId, T::CuratorGroupId, BalanceOf<T>, T::TransferId>
```

Comment:
The sender here can provide any `transfer_id`, which doesn't really make any difference, it's just a useless parameter. I think this should be refactored, so that only parameters which actually affect the result can be provided.

This can be achieved, for example, by introducing a new `ChannelTransferStatusParams` type

Reply:
How about I split `update_channel_transfer_status` into:
- `activate_channel_transfer`
- `cancel_channel_transfer`

Reply:
Sounds good, `update_channel_transfer_status` always seemed a little awkward.
I'd just suggest `initialize_channel_transfer` as a name for the first function.